### PR TITLE
feat: use hashv1 for asset files

### PR DESCRIPTION
**Pull request by nicosantangelo**
depends on https://github.com/decentraland/builder-server/pull/573

### CODE REVIEW

File: src/modules/asset/utils.ts [hunk 1]
```diff
@@ -52,8 +52,7 @@ export async function getContentsCID(asset: RawAsset): Promise<Record<string, st
 
   for (let filePath of Object.keys(contents)) {
     const file = await makeContentFile(filePath, contents[filePath])
-    const fileCID: string = await getCID([{ path: path.basename(file.path), content: file.content, size: file.size }], false)
```
Comment (LautaroPetaccio) [inline src/modules/asset/utils.ts:55]: Should we call the function getContentsHash?

File: src/modules/deployment/utils.ts [hunk 2]
```diff
@@ -42,14 +41,14 @@ export const getDefaultGroundAsset = (): Asset => ({
   actions: []
 })
 
-export async function getCID(files: ContentServiceFile[], shareRoot: boolean): Promise<string> {
+export async function getCID(files: ContentServiceFile[]): Promise<string> {
```
Comment (LautaroPetaccio) [inline src/modules/deployment/utils.ts:44]: Should we call this function getHash?

Comment (nicosantangelo) [reply src/modules/deployment/utils.ts:44]: This function didn't actually change! I just used `hasV1` when needed

File: src/modules/asset/utils.ts (hunk 1 above)
Comment (nicosantangelo) [reply src/modules/asset/utils.ts:55]: Good catch!

**Review by LautaroPetaccio — APPROVED** [review]

**Pull request merged**